### DISCUSSION TITLE
dnscrypt-proxy not start

### DISCUSSION THREAD

**Discussion by kut2tjjj** [DNSCrypt/dnscrypt-proxy#2033] [Q&A]
Windows10 1909 x64
dnscrypt-proxy 2.11 X64
RunAsAdmin service-install.bat
WINDOWS SERVIC START ERROR

**Comment by jedisct1** [comment]
Did you follow all the previous steps before [Step 5](https://github.com/DNSCrypt/dnscrypt-proxy/wiki/Installation-Windows#step-5-install-the-proxy-as-a-system-service)?

At which step did you get a different result that expected?

**Comment by kut2tjjj** [comment]
edit dnscrypt-proxy.toml file Later .dnscrypt-proxy error


Frank Denis ***@***.***> 于2022年2月9日周三 22:49写道：
